tests/farmer_harvester/test_farmer.py

```diff
@@ -908,3 +911,287 @@ def assert_pool_errors_24h() -> None:
     assert_stats_24h("stale_partials_24h")
     assert_stats_since_start("missing_partials_since_start")
     assert_stats_24h("missing_partials_24h")
+
+
+def make_pool_list_entry(overrides: Dict[str, Any]) -> Dict[str, Any]:
+    pool_list_entry = {
+        "owner_public_key": "84c3fcf9d5581c1ddc702cb0f3b4a06043303b334dd993ab42b2c320ebfa98e5ce558448615b3f69638ba92cf7f43da5",  # noqa: E501
+        "p2_singleton_puzzle_hash": "302e05a1e6af431c22043ae2a9a8f71148c955c372697cb8ab348160976283df",
+        "payout_instructions": "c2b08e41d766da4116e388357ed957d04ad754623a915f3fd65188a8746cf3e8",
+        "pool_url": "localhost",
+        "launcher_id": "ae4ef3b9bfe68949691281a015a9c16630fc8f66d48c19ca548fb80768791afa",
+        "target_puzzle_hash": "344587cf06a39db471d2cc027504e8688a0a67cce961253500c956c73603fd58",
+    }
+    for key, value in overrides.items():
+        pool_list_entry[key] = value
+    return pool_list_entry
+
+
+def make_pool_info(overrides: Dict[str, Any]) -> Dict[str, Any]:
```

Comment:
pretty minor but this function allows for overrides to be passed in, but it's only called with an empty dict I think. So you can simplify by removing the overrides

(it was flagged in coverage as not being used)

Reply:
That makes sense, done 👍